docs/HOWTOS.md

```diff
@@ -81,4 +82,51 @@ In the output of amblist, locate the user data section and find the data that th
           DATE         USER DATA
 CSECT:    EPSCMORT
           06/19/2023   MortgageApplication/a21b6ab0
-```
\ No newline at end of file
+```
+
+## Reduce necessary **application-conf** configurations within the application repository
+
+**The 3 properties for loading configuration files to configure the build framework**
+
+zAppBuild provides 3 properties `buildPropFiles`, `applicationDefaultPropFiles` and the `applicationPropFiles` that each reference a list of properties files, which contain important configuration parameters to configure the build process. The referenced properties files are loaded in the below order of precedence: 
+
+* `buildPropFiles` managed in [build-conf/build.properties](../build-conf/build.properties) references properties files in the [build-conf](../build-conf/) directory for core zAppBuild settings for the language scripts such as system datasets, naming conventions of build datasets, dataset characteristics and various core properties of the build framework as well, like the reporting features.
```

Comment:
Can we say that these properties are "enterprise-level" properties for the build framework (i.e. low-level properties)? Could you add something like that, please?